you-should-use.plugin.zsh

```diff
@@ -60,18 +60,21 @@ function _check_ysu_hardcore() {
 function _check_git_aliases() {
   if [[ "$1" = "git "* ]]; then
       local found=false
-      local tokens
+      local cmd
```

Comment:
Any particular reason you renamed this to `cmd`? At first glance `tokens` seems to describe the values of this variable better.